skillImpacts/skillImpactsModel.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../database/dbConfig');
+
+module.exports = {
+    findSkills
+};
+
+// SELECT * FROM skill_impact_requests JOIN project_goals
+// on project_goals.skill_impact_requests_id = skill_impact_requests.id
+// WHERE skill_impact_requests.campaign_id = 2;
+async function findSkills(campaign_id) {
+    const skills = await db('skilled_impact_requests')
+        .where({campaign_id:campaign_id})
```

Comment:
little JS trick: if your key and value/variable are the same like in this case, you can just write it once e.g. `{ campaign_id: campaign_id }` &rarr; `{ campaign_id }`